components/benefit_sponsors/app/domain/benefit_sponsors/operations/enrollment_eligibility/determine.rb

```diff
@@ -70,21 +70,21 @@ def find_benefit_sponsorship(benefit_sponsorship_entity_id)
         def is_initial_sponsor?(benefit_applications, effective_date)
           recent_benefit_application = benefit_applications.max_by { |benefit_application|  benefit_application.earliest_benefit_application_item.effective_period.min }
           return true unless recent_benefit_application
-          return true if recent_benefit_application.is_termed_or_ineligible? || recent_benefit_application.aasm_state == :active && recent_benefit_application.effective_period.cover?(effective_date)
+          return true if recent_benefit_application.is_termed_or_ineligible? || recent_benefit_application.aasm_state == :active && recent_benefit_application.earliest_benefit_application_item.effective_period.cover?(effective_date)
 
           ba_states = BenefitSponsors::BenefitApplications::BenefitApplication::RENEWAL_TRANSMISSION_STATES +
                       BenefitSponsors::BenefitApplications::BenefitApplication::CANCELED_STATES +
                       BenefitSponsors::BenefitApplications::BenefitApplication::EXPIRED_STATES
-          effective_date.to_date > recent_benefit_application.effective_period.max.to_date.next_day || ba_states.include?(recent_benefit_application.aasm_state)
+          effective_date.to_date > recent_benefit_application.earliest_benefit_application_item.effective_period.max.to_date.next_day || ba_states.include?(recent_benefit_application.aasm_state)
```

Comment:
```.effective_period.max.to_date.next_day```

in the query, we are trying to get an end_date so here as well we might need to get latest_benefit_application_item instead earliest_benefit_application_item